picsur/docker-compose.yml

```
@@ -31,12 +32,12 @@ services:
     user: "1000:1000"
     environment:
       PICSUR_PORT: 8286
-      PICSUR_DB_HOST: picsur-db
+      PICSUR_DB_HOST: picsur-db_1

```

@dennysubke this still needs some adaption.

The name for umbrel containers is always `<appname>_<servicename>_1`
```suggestion
      PICSUR_DB_HOST: picsur_db_1

```